cirq/google/optimizers/two_qubit_gates/gate_compilation.py

```diff
@@ -80,7 +78,7 @@ def compile_two_qubit_gate(self,
             unitaries and resulting product above.
         """
         unitary = np.asarray(unitary)
-        kak_vec = kak_vector(unitary, check_preconditions=False)
+        kak_vec = cirq.kak_vector(unitary, check_preconditions=False)
```

Comment:
The cirq convention for internal library code is to import the submodule that contains the function

```python
from cirq import linalg

linalg.kak_vector
```

Reply:
Done